x/rollup/types/events.go

```diff
@@ -3,11 +3,13 @@ package types
 const (
 	AttributeKeyL1DepositTxType   = "l1_deposit_tx_type"
 	AttributeKeyL2WithdrawalTx    = "l2_withdrawal_tx"
-	AttributeKeyFromCosmosAddress = "from_cosmos_address"
-	AttributeKeyToCosmosAddress   = "to_cosmos_address"
+	AttributeKeyFromCosmosAddress = "from_address"
+	AttributeKeyToCosmosAddress   = "to_address"
+	AttributeKeyMintCosmosAddress = "mint_address"
 	AttributeKeySender            = "sender"
 	AttributeKeyL1Target          = "target"
 	AttributeKeyValue             = "value"
+	AttributeKeyMint              = "mint"
```

Comment:
I don't love these event attribute names and feel like they may be confusing. I'll think about the best way to format our mint events and may update these